elastic/kibana
# Uptime Monitor Management - Adjust monitor details page href for browser monitors

Fixes https://github.com/elastic/kibana/issues/121549

## Summary

This PR adjusts the monitor details url in Monitor Management, specifically for browser monitors.

Depending on the type of browser monitor, inline or suite, heartbeat appends to the monitor id.

For inline monitors, the monitor id supplied by kibana is appended with `-inline`.

Suite monitors are not currently supported by Monitor Management, so we only have to support inline monitor ids. In a future version, this logic will need to be adjusted to account for suites.

### Testing

1. Connect to a hosted instance of Elasticsearch. The best way to do this is connect via `oblt-cli`.
2. Add these keys to your Kibana.yaml, including the URL for your remote Elasticsearch host. https://p.elstc.co/paste/63O1Z7Go#o1V84AgBIaU2CNNpbTWvhhSILnYMbF8BrORkxoN6Tfo
3. Navigate to `/app/uptime/add-monitor` and create a browser monitor
4. Wait for data to be indexed
5. Navigate to `/app/uptime/manage-monitors` and click on the name of the monitor to link to the monitor detail page. 
6. The link should navigate successfully to the details page for the monitor configured
